migrations/20200521135018-flatten-user-extension-entities.js

```diff
@@ -47,7 +47,7 @@ module.exports = {
       let userDocumentArray = 
       await db.collection('userExtension').find({
         _id : { $in : usersId }
-      }).project({ devices : 0 }).toArray();
+      }).project({ devices : 0, metaInformation : 0,organisations : 0, organisationRoles : 0, __v : 0 }).toArray();
```

Comment:
@aman-tunerlabs - Let's do the opposite, the project only required fields, please